Remove enkf plot_config staff from ert

xjules opened the pull request:
**Task**
_Short description of the task_
PlotConfig has been remove from libres 
here: https://github.com/equinor/libres/pull/560
We need to update PlotSettings and fix tests to work without enkf plot_config

**Pre un-WIP checklist**
- [ ] Equinor tests pass locally
- [ ] Have completed graphical integration test steps

**Depends on**
* Equinor/libres#
* Equinor/libecl#

lars-petter-hauge commented:
Ref failing tests, rebase master and you're good to go, fixed in https://github.com/equinor/ert/pull/383/commits/729a2b0bae822382e6c016a71e42006c2d2dc69e

xjules commented:
> Ref failing tests, rebase master and you're good to go, fixed in [729a2b0](https://github.com/equinor/ert/commit/729a2b0bae822382e6c016a71e42006c2d2dc69e)

Ok, rebased and the tests have passed.

ManInFez commented on ert_gui/plottery/plot_config.py:
```diff
@@ -7,11 +7,12 @@ class PlotConfig(object):
     # policy settings which can be set in the configuration file.
     
-    def __init__(self, plot_settings, title="Unnamed", x_label=None, y_label=None):
+    def __init__(self, plot_settings=None, title="Unnamed", x_label=None, y_label=None):
```
If plot_settings are not used anymore, could it not be removed from the parameter-list?

xjules replied:
I left it there in order it can be set from code if required.

markusdregi commented:
@xjules Could you bump the _libres_ version also to verify that it now builds against _libres_ after the plot config was removed?

lars-petter-hauge commented:
I suppose we need to actually run ERT as well, I don’t think there are tests which would cover this.

xjules commented:
> @xjules Could you bump the _libres_ version also to verify that it now builds against _libres_ after the plot config was removed?

Sure
